server/php-engine.cpp

```diff
@@ -542,7 +542,7 @@ int hts_func_execute(connection *c, int op) {
     return -501;
   }
 
-  vkprintf (1, "in hts_execute: connection #%d, op=%d, header_size=%d, data_size=%d, http_version=%d\n",
+  vkprintf (1, "start https execute: connection #%d, op=%d, header_size=%d, data_size=%d, http_version=%d\n",
```

Comment:
'hts' is a short of 'HTtp Server', it's not https :)

Reply:
fix